### ISSUE TITLE
Multitap not working in games that check for it after boot

### ISSUE THREAD

**Issue by r2d2rigo** [alex-free/tonyhax#65]
**tonyhax version:** 1.4.5
**Installation method:** mc annihilator + ulaunchelf on usb, on a ps2
**Entry point game:** no entry point game. just a memory card
**Console model:** SHCP-102
**Integrity check:** yes
**BIOS version:** v4.5 05/25/00 E
**Target game:** Crash Team Racing (SCES-02105)

**Bug explanation**:
Games that ask the user to plug in the multitap after being booted don't work with tonyhax (tested with Crash Team Racing, but also reproducible with Crash Bash).

When the multitap is connected, the console recognises the first controller plugged in the multitap. Connecting more than one controller crashes and reboots the console.

**Comment by alex-free** [comment]
Does this happen with v1.5.2?
Does this happen with the original tonyhax? https://github.com/socram8888/tonyhax/releases

**Comment by r2d2rigo** [comment]
@alex-free yes, it was happening with original tonyhax 1.4.5. I've updated the memory card to tonyhax international 1.5.2 and same thing happens (Crash Bash reboots the console as soon as the multitap is connected).
